### PR TITLE
Restore source code use via "download/get/*.js"

### LINKED ISSUE #199: Download URLs are not working anymore
Hi,

It looks like download URLs are not working anymore:

http://mootools.net/download/get/mootools-core-1.5.1-full-nocompat.js

We actually need to POST parameters in order to obtain a file. This is **very** unfortunate. Any chance you might fix this and provide resources that we can access with GET?

Thanks.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/6494581-download-urls-are-not-working-anymore?utm_campaign=plugin&utm_content=tracker%2F130405&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F130405&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by SergioCrisostomo**
fixes #199 

2 commits:

 - 48b457f23bb8b700f04c3f0ba433804d16596ae1 to add source files for old versions `< 1.5`
 - fc97e1b38e77ae0eb7e68106d89bd21067c1815b to add the download functionality

This PR restores a functionality that was available in the old website. To be able to download Core's source files via url `http://mootools.net/download/get/<version>.js`. This was used both by users and jsFiddle.

So basically:

 - old versions will be served as before. (example: `mootools-1.2.6-core-nc.js`, or `mootools-core-1.4.5-full-compat-yc.js`). 
 - Version starting at `1.5.0` will be compiled live and accept both Google CDNs naming*, and our own that comes with a new download from the website.

### CODE REVIEW

**Comment by SergioCrisostomo** [comment]
Refactored this PR and updated the description above.
Review input welcome.

**Comment by arian** [comment]
I'm not really sure we really miss this functionality.

**Comment by SergioCrisostomo** [comment]
@arian we were discussing yesterday in #irc that jsFiddle is broken in older versions because of this (http://jsfiddle.net/h424aykm/), and apparently some people like @olvlvl were using it also.

**Comment by olvlvl** [comment]
It's nice to have this back. Although we can use the CDN, it only serve one flavor : `.min.js`.

**Comment by SergioCrisostomo** [comment]
We talked more about this in the #IRC-dev channel and decided to not add this feature, so mootools.net will not be used as a CDN.

Google's CDN has actually different versions, just theu don't point it out. (maybe we could ask them to change that).

So for example for `1.5.1` we have:

 - compat: https://ajax.googleapis.com/ajax/libs/mootools/1.5.1/mootools.js
 - compat compressed: https://ajax.googleapis.com/ajax/libs/mootools/1.5.1/mootools-yui-compressed.js
 - no-compat: https://ajax.googleapis.com/ajax/libs/mootools/1.5.1/mootools-nocompat.js
 - no-compat compressed: https://ajax.googleapis.com/ajax/libs/mootools/1.5.1/mootools-nocompat-yui-compressed.js
